### DISCUSSION TITLE
Compiling fails for s2-mini-esp32s2_ili9341

### DISCUSSION THREAD

**Discussion by ren2342** [HASwitchPlate/openHASP#483] [Hardware]
Hello,

no matter what I do, I can't compile the code; tried it locally and with Gitpod. I uncomment the lines for the board in platforms and extra_default_envs, try to build and get this error message:

Error: Please specify platform for 's2-mini-esp32s2_ili9341' environment

Help is very much appreciated.

**Comment by fvanroie** [comment]
Please note we don't support ESP32-S2 chips anymore. They are under-powered in RAM and single-core making them less suitable for openHASP. If you tweak the configurations, you can probably make it work but this isn't recommended.

Unless you have a compelling reason to use the ESP32-S2 mini, I would advice to get an ESP32 or ESP32-S3 instead.
